packages/terra-icon/src/Icon.scss

```diff
@@ -4,28 +4,37 @@
 /**
  * IE/Edge need both width and height defined to properly size SVG icons
  */
-.terra-Icon {
-  display: inline-block;
-  vertical-align: -0.15em; // Adjusts visual alignment for icons to better align next to text
-}
 
-/**
- * The following selectors match all SVG elements, (path, cirlec, line, rect, etc), within inline SVG
- * which do not have a predefined fill color.
- */
-.terra-Icon *:not([fill]) {
-  fill: currentColor; // Allows icon color to be inherited from text color
-}
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  .icon {
+    display: inline-block;
+    vertical-align: -0.15em; // Adjusts visual alignment for icons to better align next to text
+  }
 
-@keyframes spin {
-  0% { transform: rotate(0deg); }
-  100% { transform: rotate(359deg); }
-}
+  /**
+   * The following selectors match all SVG elements, (path, cirlec, line, rect, etc), within inline SVG
+   * which do not have a predefined fill color.
+   */
+  .icon *:not([fill]) {
+    fill: currentColor; // Allows icon color to be inherited from text color
+  }
 
-.terra-Icon.is-spin {
-  animation: spin 2s infinite linear;
-}
+  :global {
+    @keyframes spin {
+      0% { transform: rotate(0deg); }
+      100% { transform: rotate(359deg); }
+    }
+  }
+
+  .icon.is-spin {
+    :global {
+      animation: spin 2s infinite linear;
+    }
+  }
 
-[dir='rtl'] .terra-Icon.is-bidi {
-  transform: scaleX(-1);
+  [dir='rtl'] .icon.is-bidi {
```

Comment:
Good question. I think in this case I do need it. I tried to rewrite this:
```
  [dir='rtl'] .is-bidi {
    transform: scaleX(-1);
  }
```
as
```
  .is-bidi {
    transform: scaleX(1);
  }
```
but it did not generate an rtl version.